src/content/docs/en/guides/integrations-guide/svelte.mdx

```diff
@@ -9,7 +9,7 @@ i18nReady: true
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 import Since from '~/components/Since.astro';
 
-This **[Astro integration][astro-integration]** enables server-side rendering and client-side hydration for your [Svelte](https://svelte.dev/) components. It supports Svelte 3, 4, and 5 (experimental).
+This **[Astro integration][astro-integration]** enables server-side rendering and client-side hydration for your [Svelte](https://svelte.dev/) 5 components. For Svelte 3 and 4 support, install `@astrojs/svelte@5` instead.
```

Comment:
> For Svelte 3 and 4 support, install `@astrojs/svelte@5` instead

Just calling attention to it because it *looks* weird. Svelte 5 is the default. If you want 3 and  install `svelte@**5**` -- just checking this isn't a typo or something!

Reply:
It's referring to install `@astrojs/svelte`, which is an integration. And `@astrojs/svelte` v5 supports Svelte 3 and 4 only.

Svelte 5 support is added on `@astrojs/svelte` v6, which I didn't mention v6 here since it will be the latest.

Reply:
Yup, figured this out in the other PR. What a pain, but understood!

Would be much nicer if our numbers aligned with theirs!

Reply:
> Would be much nicer if our numbers aligned with theirs!

Easy, just don't ship @astrojs/svelte v6 and update v5 until Svelte v6 drops. :trollface: